pkg/piecestore/psserver/config.go

```diff
@@ -17,11 +17,16 @@ var (
 
 // Config contains everything necessary for a server
 type Config struct {
-	Path                         string        `help:"path to store data in" default:"$CONFDIR/storage"`
-	WhitelistedSatelliteIDs      string        `help:"a comma-separated list of approved satellite node ids" default:""`
-	SatelliteIDRestriction       bool          `help:"if true, only allow data from approved satellites" default:"false"`
-	AllocatedDiskSpace           memory.Size   `user:"true" help:"total allocated disk space in bytes" default:"1TB"`
-	AllocatedBandwidth           memory.Size   `user:"true" help:"total allocated bandwidth in bytes" default:"500GiB"`
+	Path string `help:"path to store data in" default:"$CONFDIR/storage"`
+
+	WhitelistedSatelliteIDs string `help:"a comma-separated list of approved satellite node ids" default:""`
+	SatelliteIDRestriction  bool   `help:"if true, only allow data from approved satellites" default:"false"`
+
+	AllocatedDiskSpace memory.Size `user:"true" help:"total allocated disk space in bytes" default:"1TB"`
+	AllocatedBandwidth memory.Size `user:"true" help:"total allocated bandwidth in bytes" default:"500GiB"`
+
 	KBucketRefreshInterval       time.Duration `help:"how frequently Kademlia bucket should be refreshed with node stats" default:"1h0m0s"`
 	AgreementSenderCheckInterval time.Duration `help:"duration between agreement checks" default:"1h0m0s"`
+
+	// TTLCheckInterval time.Duration `help:"how frequently piece TTL is checked" default:"1h0m0s"`
```

Comment:
i'm a pretty big believer in not checking in commented out code. https://blog.kentcdodds.com/please-don-t-commit-commented-out-code-53d0b5b26d5f

Reply:
Anyways, removed changes in piecestore and will make a proper PR to fix them.